tests/unit_tests/output_parsers/test_base_output_parser.py

```diff
@@ -0,0 +1,43 @@
+"""Test the BaseOutputParser class and its sub-classes."""
+from abc import ABC
+from typing import List, Type
+from unittest.mock import MagicMock
+
+import pytest
+
+from langchain.schema import BaseOutputParser
+
+_PARSERS_TO_SKIP = {"FakeOutputParser", "BaseOutputParser"}
+
+
+def non_abstract_subclasses(cls: Type[ABC]) -> List[Type]:
```

Comment:
small thing, but given that this fn is fully generic aside from PARSERS_TO_SKIP, maybe we could instead add a `to_skip: Optional[Set]=None` parameter and pass _PARSERS_TO_SKIP in instead of hardcoding in this fn